apps/range-voting/app/script.js

```diff
@@ -273,17 +269,19 @@ function loadVoteSettings() {
         new Promise((resolve, reject) =>
           app
             .call(name)
-            .first()
-            .map(val => {
-              if (type === 'number') {
-                return parseInt(val, 10)
-              }
-              if (type === 'time') {
+            .pipe(
+              first(),
+              map(val => {
```

Comment:
Seems re-usable could be a custom operator. Again doesn't need to be addressed now.

Reply:
Good idea